### PR TITLE
FeatureSelection widget fixes

### PR DESCRIPTION
**Pull request by vmilan**
# Description

Shortcut: https://app.shortcut.com/cartoteam/story/321917/pending-things-to-fix-in-updated-featureselection
[sc-321917]

### CODE REVIEW

File: packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 
 const ChipList = styled(List)(({ theme: { spacing } }) => ({
   display: 'flex',
   alignItems: 'center',
   margin: 0,
-  padding: spacing(0, 1),
+  padding: 0,
```
Comment (juandjara) [inline packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js:10]: Removing this padding the storybook now looks a bit weird
Captura de pantalla 2023-06-14 a las 17 18 38

File: packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js [hunk 2]
```diff
@@ -52,19 +56,27 @@ function FeatureSelectionUIGeometryChips({
 
   function getFeatureChipLabel(feature, index) {
     const type = translateType(feature.geometry.type);
-    return feature.properties?.name || `${type} ${index + 1}`;
+    return chipLabel || feature.properties?.name || `${type} ${index + 1}`;
   }
 
+  const [onMouseElement, setOnMouseElement] = useState(false);
```
Comment (juandjara) [inline packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js:62]: Can we give a more meaningful name to this variable ?
Something like `showDeleteTooltip`

File: packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js (hunk 1 above)
Comment (vmilan) [reply packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js:10]: Fixed https://github.com/CartoDB/carto-react/pull/708/commits/42e895c5efaa33308c60f8179f15229616789a90

File: packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js (hunk 2 above)
Comment (vmilan) [reply packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js]: Fixed https://github.com/CartoDB/carto-react/pull/708/commits/42e895c5efaa33308c60f8179f15229616789a90

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Review by juandjara — APPROVED** [review]
LGTM 👍

**Pull request merged**